### ISSUE TITLE
Nested OrderBy implementation

### ISSUE THREAD

**Issue by rpaschoal** [kahanu/System.Linq.Dynamic#55]
Hello,

Any chance we can have this added to the official package? :

http://stackoverflow.com/questions/26053488/how-does-nesting-an-orderby-in-a-dynamic-linq-query-work

I want to use a nested OrderBy within a collection.

Thank you for your time.

**Comment by ryanvgates** [comment]
Can you make submit a pull request with the changes and unit tests? If everything looks good, then I can merge it in and push an updated nuget package.

**Comment by StefH** [comment]
Hi @rpaschoal ; this functionality has already been implemented in my version : https://github.com/StefH/System.Linq.Dynamic.Core

Code lines:
https://github.com/StefH/System.Linq.Dynamic.Core/blob/master/src/System.Linq.Dynamic.Core/DynamicQueryableExtensions.cs#L561

**Comment by rpaschoal** [comment]
Hello @StefH,

Yes, I found out after asking here.

Thank you for that =)

**Comment by StefH** [comment]
Cool

**Comment by ryanvgates** [comment]
@rpaschoal can you provide an example of how this would be used? I would like to add a unit test & this functionality to the package.

**Comment by rpaschoal** [comment]
Hello @ryanvgates,

I just wanted to use an "OrderBy" expression after a "Where" expression was applied.

Perhaps @StefH can give you more details about his implementation over this? I am using his port and it has been working fine for me now!

The way I use it is to build dynamic runtime expressions and queries, which can be used in, for example, a grid view where the user can apply some filtering, sorting and custom paging to a set of data. Therefore this is why I needed to apply an "OrderBy" expression (also dynamic) after whatever filter was applied through a dynamic linq expression.

**Comment by StefH** [comment]
I used this library also for that purpose : https://github.com/StefH/KendoGridBinderEx

-----Oorspronkelijk bericht-----
Van: "Rafael Carvalho" <notifications@github.com>
Verzonden: ‎17-‎10-‎2016 00:25
Aan: "kahanu/System.Linq.Dynamic" <System.Linq.Dynamic@noreply.github.com>
CC: "Stef Heyenrath" <Stef.Heyenrath@gmail.com>; "Mention" <mention@noreply.github.com>
Onderwerp: Re: [kahanu/System.Linq.Dynamic] Nested OrderBy implementation (#55)

Hello @ryanvgates,
I just wanted to use an "OrderBy" expression after a "Where" expression was applied.
Perhaps @StefH can give you more details about his implementation over this? I am using his port and it has been working fine for me now!
The way I use it is to build dynamic runtime expressions and queries, which can be used in, for example, a grid view where the user can apply some filtering, sorting and custom paging to a set of data. Therefore this is why I needed to apply an "OrderBy" expression (also dynamic) after whatever filter was applied through a dynamic linq expression.
